packages/patternfly-4/react-core/src/components/LoginPage/LoginFooterItem.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import styles from '@patternfly/patternfly-next/components/Login/login.css';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+  /** Content rendered inside the footer Link Item */
+  children: PropTypes.node,
+  /** Additional classes added to the Footer Link Item  */
+  className: PropTypes.string,
+  /** The URL of the Footer Link Item */
+  href: PropTypes.string
+};
+
+const defaultProps = {
+  children: null,
+  className: '',
+  href: '#'
+};
+
+const LoginFooterItem = ({ className, children, ...props }) => (
+  <a className={css(styles.loginFooterLink, className)} {...props}>
```

Comment:
href is not explicitly added here. also might need target="_blank" to open in new tab.
I also think that we should allow user to pass in their own Link component (e.g. react router). Maybe handle similar to here?
https://github.com/patternfly/patternfly-react/pull/991/files#diff-bd11ed69b68db9dcaeb317b19e68e785R42